docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   prometheus:
-    image: prom/prometheus:latest
+    image: docker.arvancloud.ir/prom/prometheus:latest
```

Comment:
it is a besta practice to specify version of your dependency or image. use this practice and add it to your readme why you have to do this.

Reply:
fixed